src/app/components/client/ExposureCard.module.scss

```diff
@@ -84,25 +99,46 @@
         list-style: none;
         margin-left: 0;
         padding-left: 0;
-        display: flex;
-        align-items: center;
-        gap: $layout-md;
-
-        .detailsFoundItem {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-gap: $spacing-sm;
+        
+        @media screen and (min-width: $screen-md) {
           display: flex;
+          flex-direction: row;
+          align-items: center;
+        }
 
-          svg {
-            color: $color-purple-70;
+        li + li {
+         padding-left: 23px; // offset the exposure icon
+             
+          @media screen and (min-width: $screen-md) {
+            padding-left: $layout-md;
           }
+        }
 
+        li:first-child {
+          grid-column: 1 / -1;
+        }
+
+        .detailsFoundItem {
           dt {
             display: flex;
             gap: $spacing-sm;
             align-items: center;
+            position: relative;
+
+            span {
+              color: $color-purple-60;
+              position: absolute;
```

Comment:
Why is this absolutely positioned? It looks like it should be part of the regular document flow? (And likewise for the `relative` above.)

Reply:
Ah OK, so you want the `dd` to be aligned with the heading, not the icon. In that case, I'd wrap the icon in a container with a fixed width (preferably defined in a Sass variable, to explain the magic number), and then add a `padding-inline-start` to the `<dd>` of the same size. That avoids much of the potential headaches of absolute positioning down the road, while still achieving the desired layout.